File: Problem Set 1/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change;

    do
    {
        change = get_int("Changed owed: ");
    }
    while (change <= 0);

    int coins = 0;

    coins += change / 25;
    change %= 25;

    coins += change / 10;
    change %= 10;

    coins += change / 5;
    change %= 5;

    coins += change;

    printf("%d\n", coins);
}

File: Problem Set 6/sentimental-cash/cash.py
from cs50 import get_float

change = get_float("Change: ")
while change <= 0:
    change = get_float("Change: ")

cents = round(change * 100)
coins = 0

coins += cents // 25
cents %= 25

coins += cents // 10
cents %= 10

coins += cents // 5
cents %= 5

coins += cents // 1

print(f"{coins}")
